source/core_http_client.c

```diff
@@ -1480,13 +1480,13 @@ static HTTPStatus_t writeRequestLine( HTTPRequestHeaders_t * pRequestHeaders,
     assert( methodLen != 0U );
 
     toAddLen = methodLen +                 \
-               SPACE_CHARACTER_LEN +       \
-               SPACE_CHARACTER_LEN +       \
-               HTTP_PROTOCOL_VERSION_LEN + \
-               HTTP_HEADER_LINE_SEPARATOR_LEN;
```

Comment:
Nit: Combine the values to `4U`.

Leave a comment explaining why the 4 char offset as `4U` once again seems more magical just because some of the context is lost.